tests/unit/test_xblock_poll.py

```diff
@@ -62,7 +65,23 @@ def test_student_view_user_state_handler(self):
         }
         self.assertEqual(response, expected_response)
 
+    def test_csv_export(self):
+        response = json.loads(
+            self.poll_block.handle(
+                'start_csv_export',
+                make_request('', method='POST')
+            ).body
+        )
+        expected_response = {
+            u'choice': None,
+            u'submissions_count': 5,
+            u'tally': {'R': 0, 'B': 0, 'G': 0, 'O': 0},
+        }
+
+        raise
 
+@patch('courseware.models.StudentModule')
+@patch('instructor_task.models.ReportStore')
```

Comment:
You shouldn't need to patch the existing tests... I'll explain elsewhere to move the imports so this isn't necessary.